sda-commons-web-autoconfigure/build.gradle

```diff
@@ -35,4 +35,8 @@ dependencies {
   testImplementation 'org.junit-pioneer:junit-pioneer:2.0.0', {
     exclude group: 'org.junit'
   }
+
+  test {
+    environment "METADATA_FIELDS", "tenant-id,processes"
+  }
```

Comment:
I would turn it into a configuration property and make MetadataContext a bean. Then you can define the override in the test. As this PR is not about „springification“ that's maybe not right place to do it here. But could you leave a TODO so that we can use the standard way of Spring Boot configuration later?

Reply:
As you use the `MetadataContext.metadataFields()` now, you could also add `@SetSystemProperties` to the test classes (at the very top). Example: https://github.com/SDA-SE/sda-spring-boot-commons/pull/200/files#diff-a059385d2de54b2c378821272fc0622d23ff8a57485c6af731579bef425c92ddR39